File: sample.c
#include <stdio.h>

// This program prints "Hello, World!" and does basic arithmetic
int main() {
    // Print a message
    printf("Hello, World!\n");

    // Declare two integer variables
    int a = 5;
    int b = 3;

    // Perform some basic arithmetic
    int sum = a + b;
    int difference = a - b;
    int product = a * b;
    float division = (float)a / b; // Typecasting to float for precise division

    // Print the results
    printf("Sum of %d and %d is: %d\n", a, b, sum);
    printf("Difference between %d and %d is: %d\n", a, b, difference);
    printf("Product of %d and %d is: %d\n", a, b, product);
    printf("Division of %d by %d is: %.2f\n", a, b, division);

    return 0;
}

File: sample.py
# This program prints "Hello, World!" and does basic arithmetic

def main():
    # Print a message
    print("Hello, World!")

    # Perform some basic arithmetic
    a = 5
    b = 3
    sum_result = a + b
    diff_result = a - b
    product_result = a * b
    division_result = a / b

    # Print the results
    print(f"Sum of {a} and {b} is: {sum_result}")
    print(f"Difference between {a} and {b} is: {diff_result}")
    print(f"Product of {a} and {b} is: {product_result}")
    print(f"Division of {a} by {b} is: {division_result:.2f}")

# Call the main function
if __name__ == "__main__":
    main()
